src/mame/drivers/ms0515.cpp

```diff
@@ -209,36 +442,126 @@ PALETTE_INIT_MEMBER(ms0515_state, ms0515)
 	palette.set_pen_color(15, rgb_t(255, 255, 255));
 }
 
-WRITE8_MEMBER(ms0515_state::ms0515_portc_w)
+// from vt240.cpp
+void ms0515_state::irq_encoder(int irq, int state)
+{
+	if(state == ASSERT_LINE)
```

Comment:
Spacing isn't consistent with the rest of the file here - there's no space after the `if` in this function while there is everywhere else.